README.md

```diff
@@ -58,6 +58,13 @@ In local network mode, you can optionally pass:
    - `testnet` sets limits to match those used on testnet (the default quickstart configuration)
    - `unlimited` sets limits to the maximum resources that can be configured
 
+- `--enable {core, horizon, rpc}` to further select which services are started in the quickstart container. Since using a 'local' network, the core service will be started regardless as it runs the 'local' network, and these additional permutations of `--enable` are applicable:
+   - not specified, the default behavior will be to run core, horizon, friendbot, rpc. 
+   - `core` only runs the stellar core and an associated archive server.
+   - `core,horizon` runs core, horizon, friendbot
+   - `horizon` runs core, horizon, friendbot  
+   - `rpc` runs core, horizon, friendbot, rpc. 
+
```

Comment:
The options listed in this section as options not available on other modes. Instead of adding this here, can you add the description of what's unique in local mode to the section about selecting services, it's one section below under the heading "Service Options".
```suggestion
__Note: The `--enable` options behaves differently in local network mode, see [Service Options](#service-otions) for more details.__
```